plugins/AudioFileProcessor/AudioFileProcessorWaveView.cpp

```diff
@@ -467,9 +469,11 @@ void AudioFileProcessorWaveView::reverse()
 			- m_sample->endFrame()
 			- m_sample->startFrame()
 	);
+	
+	const int m_from_ = m_from;
```

Comment:
I'd prefer a name like `fromBefore` or `fromTmp` or similar. `m_` implies that it's a member which is confusing because it is a local variable and the `_` at the end also looks a bit off.

Reply:
sure 😅 that's my python habits creeping in, sorry

Reply:
Some habits are hard to shake off. 😅